packages/builder/src/components/design/settings/controls/ValidationEditor/ValidationDrawer.svelte

```diff
@@ -65,6 +65,14 @@
       label: "Must not contain",
       value: "notContains",
     },
+    MaxFileSize: {
+      label: "Max file size (MB)",
+      value: "maxFileSize",
+    },
+    MaxUploadSize: {
+      label: "Max upload size (MB)",
```

Comment:
Could we change this to `Max total upload size (MB)` just to make it slightly clearer that this represents the total of all files rather than a single one?